virtcontainers/sandbox.go

```diff
@@ -1600,10 +1602,10 @@ func (s *Sandbox) HotplugAddDevice(device api.Device, devType config.DeviceType)
 			if _, err := s.hypervisor.hotplugAddDevice(dev, vfioDev); err != nil {
 				s.Logger().
 					WithFields(logrus.Fields{
-						"sandboxid":       s.id,
-						"vfio device ID":  dev.ID,
-						"vfio device BDF": dev.BDF,
-					}).WithError(err).Error("failed to hotplug VFIO device")
+					"sandboxid":       s.id,
+					"vfio device ID":  dev.ID,
```

Comment:
Again, I think it's best to avoid field names with spaces "just in case", so maybe `vfio-device-id` ;)